### DISCUSSION TITLE
Support editing status emoji/text

### DISCUSSION THREAD

**Discussion by ishepherd** [community/community#33678] [Mobile] [answered]
Hi, I am on vacation ⛱️ and I would like to reflect that in my GitHub status emoji/text.

It seems like it would be a natural feature for mobile but I can't find it (Android)

**Comment by cosmicqbit** [comment, marked as answer]
If you can't find it in native app, try logging in from browser.

**Comment by d4kw1n** [comment]
If you use the application, this function is not available at the moment. You can use github on the phone website to do this. Wish you have a nice vacation 🌴

**Comment by ishepherd** [comment]
Thanks everyone I have successfully changed on the mobile website.

I have moved this thread to "Mobile" category. It wasn't clear that selecting Profile category would deselect Mobile ...

can only change category in the mobile website. So that's another feature ask 😜

**Reply by cosmicqbit** [reply]
I'm glad that you figured it out. Mark [this](https://github.com/orgs/community/discussions/33678#discussioncomment-3698039) as answered if it was helpful. Enjoy your vacation 😊❤️
